andela/team-odd-bn-backend
# #170799329 Protect all endpoints

#### What does this PR do?
The PR helps to protect all backend endpoints.

#### Description of Task to be completed?
**Current**
- Blacklisted token are passing
- Two different middleware token verification are used

**Expected**
- Reject blacklisted token
- Use one middleware for token verification

#### How should this be manually tested?
- Clone this repository using the command: `git clone https://github.com/andela/team-odd-bn-backend.git`
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout -b bg-protect-backend-routes-
170799329
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170799329](https://www.pivotaltracker.com/story/show/170799329)

#### Screenshots (if appropriate)

User try to access notification with blocked token
<img width="1440" alt="Screen Shot 2020-01-20 at 15 54 46" src="https://user-images.githubusercontent.com/56690376/72732030-8ece2400-3b9d-11ea-8fa2-a48dc4a35dcd.png">
